http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/DateAccessedUpdater.scala

```diff
@@ -45,21 +45,23 @@ class DateAccessedUpdater[F[_]](
       .compile
       .drain
 
-  private def updateDateAccessed(msg: UpdateDateAccessMessage): F[Unit] =
-    metrics.incrementCounter("jupyterAccessCount") >>
-      clusterQuery
-        .clearKernelFoundBusyDateByProjectAndName(msg.cloudContext, msg.runtimeName, msg.dateAccessd)
-        .flatMap(_ =>
-          clusterQuery.updateDateAccessedByProjectAndName(msg.cloudContext, msg.runtimeName, msg.dateAccessd)
-        )
-        .transaction
-        .void
+  private def updateDateAccessed(msg: UpdateDateAccessMessage): F[Unit] = msg.updateTarget match {
+    case UpdateTarget.Runtime(runtimeName) =>
+      metrics.incrementCounter("jupyterAccessCount") >>
+        clusterQuery
+          .clearKernelFoundBusyDateByProjectAndName(msg.cloudContext, runtimeName, msg.dateAccessd)
+          .flatMap(_ => clusterQuery.updateDateAccessedByProjectAndName(msg.cloudContext, runtimeName, msg.dateAccessd))
+          .transaction
+          .void
+    case UpdateTarget.App(appName) =>
+      appQuery.updateDateAccessed(appName, msg.dateAccessd).transaction.void
```

Comment:
Maybe add an `appAccessCount` metric?

Reply:
I was thinking the jupyter one isn't very useful, so didn't add one for app either...are you thinking it'd be good to just see the overall traffic?